### PR TITLE
Email confirmation template

### LINKED ISSUE #21: Email sender template
Create email template for send verification code on email. (6 digits)

> **Useful:** 
> Don`t use flex grid. Check what u can use in google

### PR DESCRIPTION
**Pull request by maria2908**
I made email confirmation template, please review it

### CODE REVIEW

File: StudCity.Frontend/src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ function App() {
     return (
         <div className="App">
             <header className="App-header">
+
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/App.js]: remove paragraph please

File: StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div style="max-width: 550px; width: 100%; margin: 0 auto; text-align:center; border-radius: 31% 69% 51% 49% / 62% 30% 70% 38% ; background-color:darkcyan">
+    <img style="height: 450px; margin: 0;" src="https://niceillustrations.com/wp-content/uploads/2022/01/Verification-Code-color-800px.png" alt="">
+        <h1 style="margin: 0;">Welcome budy!</h1>
+    <p style=" font-family: 'Hind', sans-serif; font-weight: 600; margin: 20px 0; font-size: 18px;">
+    You registered an account on Stud-city, before being able to use your account you need to verify your email. Here is your verification code:
+    </p>
+    <div style='display: flex; justify-content: center; font-weight: 500; color:black; font-size: 30px;  margin-bottom: 20px;'>
```
Comment (Serhiy-dotcom) [inline StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html:7]: replace single quotes with double quotes

File: StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div style="max-width: 550px; width: 100%; margin: 0 auto; text-align:center; border-radius: 31% 69% 51% 49% / 62% 30% 70% 38% ; background-color:darkcyan">
+    <img style="height: 450px; margin: 0;" src="https://niceillustrations.com/wp-content/uploads/2022/01/Verification-Code-color-800px.png" alt="">
+        <h1 style="margin: 0;">Welcome budy!</h1>
+    <p style=" font-family: 'Hind', sans-serif; font-weight: 600; margin: 20px 0; font-size: 18px;">
```
Comment (Serhiy-dotcom) [inline StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html]: remove a space before font-family

File: StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div style="max-width: 550px; width: 100%; margin: 0 auto; text-align:center; border-radius: 31% 69% 51% 49% / 62% 30% 70% 38% ; background-color:darkcyan">
+    <img style="height: 450px; margin: 0;" src="https://niceillustrations.com/wp-content/uploads/2022/01/Verification-Code-color-800px.png" alt="">
```
Comment (Serhiy-dotcom) [inline StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html]: add a value to alt attribute

File: StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <p style=" font-family: 'Hind', sans-serif; font-weight: 600; margin: 20px 0; font-size: 18px;">
+    You registered an account on Stud-city, before being able to use your account you need to verify your email. Here is your verification code:
+    </p>
+    <div style='display: flex; justify-content: center; font-weight: 500; color:black; font-size: 30px;  margin-bottom: 20px;'>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{0}</p>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{1}</p>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{2}</p>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{3}</p>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{4}</p>
+        <p style="padding: 15px; background-color: #e9e8e8; border-radius: 10px; margin: 0 15px 0 0; ">{5}</p>
```
Comment (Serhiy-dotcom) [inline StudCity.Backend/StudCity.API/Templates/ConfirmationLetter.html]: and some spaces above at the end of style attribute value

**Review by Serhiy-dotcom — CHANGES_REQUESTED** [review]

**Review by maryanchuk04 — APPROVED** [review]
Fix Serhio comments and send request again) 

I'm waiting...

**Review by maryanchuk04 — APPROVED** [review]
You can merge this PR.
Great job.

**Review by Serhiy-dotcom — APPROVED** [review]
Great work, go ahead and merge!

**Pull request merged**